packages/MarkdownEditor-Tests.package/MarkdownBlockParserTest.class/instance/testHeadingFollowedByParagraphWithInlineEmphasis.st

```diff
@@ -0,0 +1,8 @@
+tests - text with inline
+testHeadingFollowedByParagraphWithInlineEmphasis
+
+	self
```

Comment:
Many of these tests are kinda redundant, are they? The runs are checked somewhere else.

Reply:
Which of redundancy do you mean?
The tests under `tests - text with inline` are meant as acceptance tests (they get quite complicated), so they of course test already covered features by unit tests.

Reply:
The tests in `tests - text with inline` are meant as acceptance test.
That said, they will test more complex situations which should already be covered by unit tests.

Reply:
I think in general we should ask somebody about redundancy in integration and acceptance tests. It's pretty common in our tests by now but I personally think that it makes sense because the integration and acceptance tests give you a better idea of how a class is used than having to go through all the tiny unit tests. As I said this is only my opinion. I don't know what the best practice is, but I think we should find out and then follow the best practice.